common/tools/hextobin/build.sh

```diff
@@ -15,7 +15,7 @@ cd "$THIS_SCRIPT_PATH"
 builder_describe "Build hextobin" clean configure build
 builder_describe_outputs \
   configure /node_modules \
-  build     build/index.js
+  build     /common/tools/hextobin/build/index.js
```

Comment:
this belongs in #9676 but I don't know if it's worth moving it, as it is only cosmetic for performance